src/pages/JoinComplete/JoinComplete.tsx

```
@@ -0,0 +1,41 @@
+import CenterLogo from "./JoinComplete.Centerlogo";
+import { MessageCircle } from "lucide-react";
+import { Button } from "@/components/ui/button";
+
+export default function JoinComplete() {
+    return (
+
+    <AlignCenter>
+        
+        <Wrapper>
+            <Confetti recycle={false} numberOfPieces={1000} />
+            <h1 className="text-4xl font-bold tracking-tight">가입을 축하합니다! 🎉</h1>
```

Biome (1.9.4)

[error] These CSS classes should be sorted.

Safe fix: Sort the classes.


(lint/nursery/useSortedClasses)